### PR TITLE
Create contributing guidelines.

### PR DESCRIPTION
**Pull request by avitorio**
This PR refers to #58

It covers: 
- Initial CONTRIBUTING.md file
- Initial pull_request_template.md
- Addition to contributing link to the README

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
+# Contributing to Outstatic
+
+- Before jumping into a PR be sure to search [existing PRs](https://github.com/avitorio/outstatic/pulls) or [issues](https://github.com/avitorio/outstatic/issues) for an open or closed item that relates to your submission.
+- If you can't get something to work the way you expect or found a bug, [open a question](https://github.com/avitorio/outstatic/discussions/new?category=q-a) in our discussion forums.
```
Comment (jakobo) [inline CONTRIBUTING.md:4]: Proposed text:
> If you can't get something to work the way you expect, [open a question](https://github.com/avitorio/outstatic/discussions/new?category=q-a) in our discussion forums.

If someone accidentally opens an issue that should be a discussion, it's one click for us to move the item over to the discussion forum.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+- If you are interested in adding a feature or improving the code, please [start a discussion](https://github.com/avitorio/outstatic/discussions/new?category=ideas) about it first.
+
```
Comment (jakobo) [inline CONTRIBUTING.md:6]: Suggestion: Explain the package structure (what's in /apps, vs /package vs /example)

**Review by jakobo — APPROVED** [review]
This looks good. Any comments included are just minor quibbles and are not required to be resolved.

File: CONTRIBUTING.md (hunk 2 above)
Comment (avitorio) [reply CONTRIBUTING.md:19]: Done

**Pull request merged**